packages/ui/src/Button/Button.styles.ts

```diff
@@ -0,0 +1,16 @@
+import { css } from "emotion";
+
+/**
+ * Controls the button styles
+ * @type {string}
+ */
+const webinyButtonStyles = css(
+    {},
+    {
+        "&.webiny-ui-button--secondary:not(:disabled)": {
+            borderColor: "var(--mdc-theme-primary)"
+        }
+    }
+);
```

Comment:
@endymion1818 This will look much nicer if you use the template literals; then you also don't need to use the camelCase for properties. You just write plain CSS instead.
```
const webinyButtonStyles = css`
    &.webiny-ui-button--secondary:not(:disabled) {
        border-color: var(--mdc-theme-primary)
    }
`
```

Reply:
I was following what I thought was a convention but very happy you said this, I prefer template literals too.